packages/api/models/granules.js

```diff
@@ -31,9 +31,32 @@ const granuleSchema = require('./schemas').granule;
 
 class Granule extends Manager {
   constructor() {
+    const globalSecondaryIndexes = [{
```

Comment:
I think this is a dumb question, but is there any way we can avoid have to repeat the index name in the constructor, in `getGranulesForCollection`, and in the YAML? I don't really think this is necessary to change, just wondering.

Reply:
The `index` in the constructor is for the local stack testing when `create granules table`, the actual table is created by YAML.  I'm not sure how these two can be coordinated.